org.splash.Splash.json

```diff
@@ -0,0 +1,102 @@
+{
+    "app-id": "org.splash.Splash",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "splash",
+    "rename-desktop-file": "splash.desktop",
+    "rename-icon": "splash",
```

Comment:
Not a blocker: you should probably rename these two upstream too. See the freedesktop specs https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#file-naming

Reply:
Thanks for the link, I understand better how the Flatpak packages are named now. I have a question though: Splash does not have a website of its own (yet), so as told in this documentation I should use the name `ca.sat.Splash` (SAT being the organization which supports the software).

My question is: is it possible to rename the package afterwards, once a domain name has been bought for the software?

Reply:
If it's doesn't have a website for now, you can use com.gitlab.$username.$projectSlug or the equivalent if the code is hosted elsewhere as you need to "own" that domain name. If and once you have a domain name, the package can be renamed of course. 

Anyway, as you don't own `splash.org` you will need to update the app-id to something that you actually own before we can merge the app.

Reply:
Alright, I'll goo with the com.gitlab solution for now, thanks.